eng/pipelines/aggregate-reports.yml

```diff
@@ -35,6 +35,14 @@ stages:
                 -WorkingDirectory '$(Build.ArtifactStagingDirectory)'
                 -NupkgFilesDestination 'nupkgFiles'
 
+          - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@2
```

Comment:
What's this do?

Reply:
See https://github.com/Azure/azure-sdk-tools/issues/7085

Reply:
After reading the linked docs over a couple of times, I still have no idea what an HPV is or what this actually does - but the take-away is "we have to do this," so I guess that's enough to sate my curiosity.